apps/nextjs/src/app/api/chat/fixtures/FixtureRecordLLMService.ts

```diff
@@ -0,0 +1,73 @@
+import { Message } from "@oakai/aila";
+import { LLMService } from "@oakai/aila/src/core/llm/LLMService";
+import { OpenAIService } from "@oakai/aila/src/core/llm/OpenAIService";
+import fs from "fs/promises";
+import { ZodSchema } from "zod";
+
+export class FixtureRecordLLMService implements LLMService {
+  name = "FixureRecordLLM";
+  public fixtureName: string;
+  private _openAIService: OpenAIService;
+
+  constructor(fixtureName: string, chatId: string) {
+    this.fixtureName = fixtureName;
+    this._openAIService = new OpenAIService({ userId: undefined, chatId });
+  }
+
+  async createChatCompletionStream(params: {
+    model: string;
+    messages: Message[];
+    temperature: number;
+  }): Promise<ReadableStreamDefaultReader<string>> {
+    return this._openAIService.createChatCompletionStream(params);
+  }
```

Comment:
I'm a bit confused about the distinction between these two methods. It looks like only one is used?

Reply:
Sorry which two methods?

Reply:
createChatCompletionStream and createChatCompletionObjectStream

It now looks like post-toggle, they represent whether we're using structured outputs